.eslintrc.yml

```diff
@@ -0,0 +1,112 @@
+parser: "@typescript-eslint/parser"
```

Comment:
@Min2who is this needed here? I guess TypeScript and related technologies are not to be used in this PR.

Reply:
i found the typescript parser to be best among the babel and ecma parser

Reply:
I agree this feels unexpected. We are not using Typescript (yet) in this project, so I expect all JS files to fulfill with standard vanilla JS